doc/source/basics.rst

```diff
@@ -1725,6 +1725,7 @@ pass ``convert_numeric=True``. This will force strings and numbers alike to be n
 they will be set to ``np.nan``.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
This one is for the deprecation of `convert_objects`. But actually, this whole part should be updated to use the new `to_numeric`